package.json

```diff
@@ -17,6 +17,9 @@
         "@emotion/react": "^11.11.4",
         "@emotion/styled": "^11.11.0",
         "@fontsource/roboto": "^5.0.12",
+        "@fortawesome/fontawesome-svg-core": "^6.6.0",
+        "@fortawesome/free-solid-svg-icons": "^6.6.0",
+        "@fortawesome/react-fontawesome": "^0.2.2",
```

Comment:
What are these used for?

Reply:
they were used in a previous code i wrote and referenced online